src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -4376,6 +4376,8 @@ emit_wasm_bitoperations_intrinsics (MonoCompile *cfg, MonoMethod *cmethod, MonoM
 	switch (id) {
 		case SN_LeadingZeroCount:
 		case SN_TrailingZeroCount: {
+			if (!MONO_TYPE_IS_PRIMITIVE (fsig->params [0]))
+				return NULL;
 			return emit_wasm_zero_count(cfg, fsig, args, cmethod->klass, id, arg0_type);
```

Comment:
do we need to do the same thing in the other places?

Reply:
No idea, this is the one i ran into.

Reply:
the same call is made a few lines down which made me wonder